scripts/kokoro/nabra/add_meta_data.py

```
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+# Stamp sherpa-onnx Kokoro metadata onto a Nabra-82M ONNX model and bake in
+# the iSTFT image-tone notch FIR.
+#
+# Usage: python3 add_meta_data.py <nabra.onnx> <output.onnx>
+import sys
+
+import numpy as np
+import onnx
+from onnx import TensorProto, helper, numpy_helper
```

Flake8 (7.3.0)

[error] 'onnx.TensorProto' imported but unused

(F401)